content/blog/announcing-enum-support/index.md

```diff
@@ -0,0 +1,285 @@
+
+{{< chooser >}}
+{{< choosable language typescript >}}
+
+```typescript
+import * as pulumi from "@pulumi/pulumi";
+import { s3 } from "@pulumi/aws";
+
+const myBucket = new s3.Bucket("myBucket", { acl: s3.CannedAcl.Private });
+```
+
+{{< /choosable >}}
+{{< choosable language python >}}
+
+```python
+import pulumi
+from pulumi_aws import s3
+
+my_bucket = s3.Bucket("myBucket", acl=s3.CannedAcl.PRIVATE)
+```
+
+{{< /choosable >}}
+{{< choosable language csharp >}}
+
+```csharp
+using Pulumi;
+using S3 = Pulumi.Aws.S3;
+
+class MyStack : Stack
+{
+    public MyStack()
+    {
+        var myBucket = new S3.Bucket("myBucket", new S3.BucketArgs
+        {
+            Acl = S3.CannedAcl.Private,
+        });
+    }
+}
+```
+
+{{< /choosable >}}
+{{< choosable language go >}}
+
+```go
+package main
+
+import (
+    "github.com/pulumi/pulumi-aws/sdk/v3/go/aws/s3"
+    "github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+)
+
+func main() {
+    pulumi.Run(func(ctx *pulumi.Context) error {
+        myBucket, err := s3.NewBucket(ctx, "myBucket", &s3.BucketArgs{
+            Acl: s3.CannedAclPrivate,
+        })
+        if err != nil {
+            return err
+        }
+        return nil
+    })
+}
+```
+
+{{< /choosable >}}
+{{< /chooser >}}
+
+## IDE Superpowers
+
+Enum properties provide discoverable and normalized constants that can be used in place of raw strings, allowing you to move faster with the added boosts of IDE tooling like type hints and autocomplete.
+
+Using the provided constants, you can avoid referring back to the documentation to remember the valid values and save precious moments in the development cycle that would be lost to debugging errors caused by typos.
+
+{{< chooser >}}
+{{< choosable language typescript >}}
+
+![ENUM_TYPESCRIPT](ts-enum.gif)
+
+{{< /choosable >}}
+{{< choosable language python >}}
+
+![ENUM_PYTHON](python-enum.gif)
+
+{{< /choosable >}}
+{{< choosable language csharp >}}
+
+![ENUM_CSHARP](csharp-enum.gif)
+
+{{< /choosable >}}
+{{< choosable language go >}}
+
+![ENUM_GO](go-enum.gif)
+
+{{< /choosable >}}
+{{< /chooser >}}
+
+## Optimized for flexibility
+
+While some properties make sense as "strict" enums (i.e., the input value **must** be one of the enumerated values), there are times when it would be beneficial to have constants for commonly used values without restricting the input to **only** those values. We call this second category "relaxed" enums, and we model them a little differently in our schema.
+
+### "Strict" enums
+
+A property is a "strict" enum when the input value **must** be one of the enumerated values. In this case, the property type is specified as the enum type. Currently, only the Azure NextGen provider uses "strict" enums as the OpenAPI specification explicitly defines its properties as such.
```

Comment:
Perhaps leave this detail to the section where you talk about where enums are currently supported?  

The note to add here might be to highlight that we will use "strict" enums when we are sure that the enum will include all legal values, such as when the provider is auto-generated from a specification provided by the cloud provider - like for our Azure NextGen or Kubernetes providers (with links to blog posts on how these work?).